contracts/manager/CrunaManager.sol

```diff
@@ -414,18 +426,30 @@ contract CrunaManager is Actor, CrunaManagerBase {
     return (active, disabled);
   }
 
+  /// @dev Internal function to disable a plugin but index and key
```

Comment:
The last comment on this issue as there seems to be quite a lot, so there must be a reason.